.gitignore

```diff
@@ -2,3 +2,4 @@
 /node_modules/
 /npm-debug.log
 .env
+.vs/
```

Comment:
you probably need to add this to your `.gitignore_global` file @chris--young 

I don't mind it being comitted here as well, though.